jms-client-sink/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+sink.url=amqps://remote.bouvetinterchange.eu
```

Comment:
Can this be omitted from src/main when you have the environment variables mentioned in jms_client_remote_sink.sh?